rfcs/simplify_license.md

```diff
@@ -0,0 +1,49 @@
+## RFC #18: Simplify License
+
+### Summary
+
+Reduce WPT's license to the 3-clause BSD license and update the copyright
```

Comment:
3-Clause BSD License*

Reply:
This wasn't fixed (the casing).

Reply:
Oh, sorry about that! This patch mentions the license by name twice; I didn't realize that and marked your comment as "resolved" [when I corrected the second](https://github.com/web-platform-tests/rfcs/pull/18/commits/87af2f55807d50b8a0d640d0cd564b7f3e6af4b0)